lib/astutils.cpp

```diff
@@ -374,7 +374,8 @@ bool isVariableDecl(const Token* tok)
         return false;
     if (var->nameToken() == tok)
         return true;
-    if (Token::Match(var->declEndToken(), "; %var%") && var->declEndToken()->next() == tok)
+    const Token * const varDeclEndToken = var->declEndToken();
```

Comment:
isn't this something that a compiler can optimise automatically?

Reply:
no it was not a trivial getter as I thought

Reply:
That's why I refactored out the repeated calls. Just saw it by accident in a profile session.